Connection.class.php

```diff
@@ -74,7 +74,7 @@ public function close() {
      */
     public function query($sql) {
         $this->connect();
-        $result = parent::query($sql);
+        $result = parent::query(mysql_escape_string($sql));
```

Comment:
hey @studentmsc2018,
first of all, thanks for your contribution.

according to [PHP docs](http://php.net/manual/en/function.mysql-escape-string.php), `mysql_escape_string` is deprecated.  you should use `mysqli::escape_string`. for this case `self::escape_string` should work fine.

could you please apply the changes?

Reply:
you are right. I prepared a new commit commit 0dfe313

Reply:
you are right. I prepared a new commit commit 0dfe313